modules/apps/object/object-web/src/main/resources/META-INF/resources/js/components/ObjectRelationship/ObjectRelationshipFormBase.tsx

```diff
@@ -143,38 +176,127 @@ export function ObjectRelationshipFormBase({
 		});
 	}, [creationLanguageId, objectDefinitions, query]);
 
+	const switchObjects = () => {
+		const previousObjectDefinition1 = {
+			...objectDefinition1,
+		};
+
+		setObjectDefinition1(objectDefinition2);
+
+		setObjectDefinition2(previousObjectDefinition1);
+	};
+	const handleReverseOrder = () => {
```

Comment:
@rebsilva Here we need the blank line after declaring `switchObjects` because it's used inside `handleReverseOrder` :)